### ISSUE TITLE
404 for all routes

### ISSUE THREAD

**Issue by tpoxa** [kraken-php/demo-chat#1]
I managed to start it finally, but for all routes I have 404 error. Don't know why.

PHP 7.2.0-dev (cli) (built: Oct 10 2016 16:22:13) ( ZTS DEBUG )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.1.0-dev, Copyright (c) 1998-2016 Zend Technologies

My changes was only replacing in SocketListener  ip address, I made it listed external ipaddress. 

Kraken 0.3.1 by Kamil Jamroz and contributors.
Executing : project:create ... success!
Response  : Project has been created.

**Comment by khelle** [comment]
Did you change the ip address in `public/js/index.js' also?

**Comment by zhiephie** [comment]
@khelle yeah I too, try change index.js listen to external ip, what is wrong?

**Comment by khelle** [comment]
Could you please link me the changes you made, so I can try to detect the problem?

**Comment by zhiephie** [comment]
I change localhost:6080 to ip:6080, and run server and client again. what's wrong with the way I? :smile:

**Comment by khelle** [comment]
I suspect this is the problem with origin or ip filtering. If you provide me your changed files I could ensure that everything is configured properly and the bug is on the kraken side, so then I would be able to run tests and fix it.

**Comment by zhiephie** [comment]
I'm only change `this.connection.ws = new WebSocket("ws://localhost:6080/chat");` to `this.connection.ws = new WebSocket("ws://192.168.1.164:6080/chat");` and restart kraken server and kraken client. 

and i'm visit link 192.168.1.164:6080 not working.

**Comment by tpoxa** [comment]
Sorry. I left this environment. I unable to recreate it.

**Comment by khelle** [comment]
@zhiephie I believe lines 157-159 in `Kraken\Netwok\Http\Component\Router\HttpRouter` might be an issue. Could you try changing them from:

```
$this->checkOrigin ? [] : [ 'Origin' => $this->host ],
[],
$this->checkOrigin ? '' : $this->host
```

to

```
!$this->checkOrigin ? [] : [ 'Origin' => $this->host ],
[],
!$this->checkOrigin ? '' : $this->host
```

And tell me if this helped?

**Comment by zhiephie** [comment]
@khelle Thanks worked :smile:

**Comment by khelle** [comment]
Glad to hear that. I will provide these fixes to `0.3.2` release, but until then you have to keep these lines changed in your instance for it to work properly :)
